### PR TITLE
Improve party-robot hints.md (#1588)

### PR DESCRIPTION
**Pull request by kahgoh**
* Add hints for each task

* Replace string concatenation with link to documentation

### CODE REVIEW

File: exercises/concept/party-robot/.docs/hints.md [hunk 1]
```diff
@@ -3,17 +3,22 @@
...
-- To add one string to another, you can use the `"+="` assignment.
+## 1. Welcome a new guest to the party
 
-```go
-s := "Hello"
-s += " world."
-fmt.Println(s)
-// Output: Hello world.
-```
+- `fmt`s `Sprintf` lets you use a [template to create a string][Sprintf].
```
Comment (jmrunkle) [inline exercises/concept/party-robot/.docs/hints.md:10]:
```suggestion
- `fmt.Sprintf` lets you use a [template to create a string][Sprintf].
```

Seems a little clearer in my opinion.

File: exercises/concept/party-robot/.docs/hints.md [hunk 2]
```diff
@@ -3,17 +3,22 @@
...
 
-- The [Go 101 string section][go101] has more details about this concept.
+## 2. Welcome a new guest to the party whose birthday is today
+
+- `Sprintf` can interpolate [more than one value into your string][Sprintf-multiple-values]!
```
Comment (jmrunkle) [inline exercises/concept/party-robot/.docs/hints.md:14]:
```suggestion
- `fmt.Sprintf` can interpolate [more than one value into your string][Sprintf-multiple-values]!
```

We should be consistent.

**Review by jmrunkle — APPROVED** [review]
Mind making these small changes?

**Comment by kahgoh** [comment]
> Mind making these small changes?

Sounds good. Just applied the updates.

**Comment by kahgoh** [comment]
> Mind making these small changes?

Sounds good. I've just applied the updates.

**Pull request merged**